client/src/components/categorical/value/index.js

```diff
@@ -494,8 +615,8 @@ class CategoryValue extends React.Component {
               display={isColorBy && categories ? "auto" : "none"}
               style={{
                 marginLeft: 5,
-                width: 11,
-                height: 11,
+                width: VALUE_HEIGHT,
```

Comment:
Since this is inside the `value/index.js` component, it could also be `this.height` as a constant set in the constructor

Reply:
I'm going to opt-out of doing this to consolidate all the dimension constants in one location